zombie-monitor/src/main/scala/com/broadinstitute/dsp/zombieMonitor/DbReader.scala

```diff
@@ -55,10 +55,13 @@ object DbReader {
            update PERSISTENT_DISK set status = "Deleted", destroyedDate = now() where id = $id
            """.update
 
-  def updateK8sClusterStatusQuery(id: Int) =
+  def markK8sClusterDeletedQuery(id: Int) =
     sql"""
-           update KUBERNETES_CLUSTER set status = "DELETED", destroyedDate = now() where id = $id
-           """.update
+          UPDATE NODEPOOL
```

Comment:
oh, didn't realize it needs to join so many tables..I probbly would've been fine with what you had before :) ...